src/Network/Network/BYOIP/CustomIpPrefix/NewAzureCustomIpPrefixCommand.cs

```diff
@@ -166,10 +166,6 @@ private PSCustomIpPrefix CreateCustomIpPrefix()
                 {
                     psModel.PrefixType = "Child";
                 }
-                else
-                {
-                    psModel.PrefixType = "Singular";
-                }
```

Comment:
Hi, in this code, psModel.PrefixType could be empty, is that right? If it is, I'll merge the pr.

Reply:
Hi, that is correct. For compatibility reasons, the PrefixType field needs to be empty in the default case.